CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
+- Changed: use ``react-router`` basename feature
+  ([#165](https://github.com/MoOx/statinamic/pull/165))
 - Changed: build and client scripts now require an exports value
   that must contains layouts, metadata and routes.
   This values should be strings.
   See changes in ``boilerplate/scripts/build.js``
   ([#145](https://github.com/MoOx/statinamic/pull/145))
+- Fixed: devServer should redirect you to the right page when url ends with
+  ``index.html`` instead of showing a error page
+  ([#165](https://github.com/MoOx/statinamic/pull/165))
```

Comment:
Yup. Trying to handle this in development mode. using url with index.html will cause invalid checksum